### DISCUSSION TITLE
hasmany relation: how to show a table with ALL items instead of only those that are already "associated" ?

### DISCUSSION THREAD

**Discussion by michabbb** [filamentphp/filament#5583] [Help]
when it comes to "Associate" and "Dissociate" i only see the item that are already associated to the resource.
but I would like to have table, with ALL items of the other resource so that I can "bulk associate" multiple items
by just clicking on a checkbox in the table below instead of opening 100 times a modal window.

so, instead of:

10 x click on "Associate" -> search -> click

I would prefer to see in the table below "all items that exist"  
and just hit some checkboxes (like the bulk-checkbox) and just hit once "Associate"  
and with that, many resources got "associated", in bulk.

is that possible ? 😏 🤔 

![image](https://user-images.githubusercontent.com/3524595/215566915-a3790e03-018c-4f2f-99e1-30f1f7799ffe.png)

**Comment by danharrin** [comment]
Make a Bulk Action on the post resource which sets the category instead

**Reply by michabbb** [reply]
for someone who is new to filament, i would be very thankful if you could provide me one or two links from the documentation, that lead me in the right direction. 🙏

**Reply by danharrin** [reply]
https://filamentphp.com/docs/2.x/tables/actions#custom-forms

**Comment by michabbb** [comment]
@danharrin for my understanding: where exactly to you place that bulk-action in my example ? do you suggest to go to the "posts" resource (where I see all posts) and make it there ? if yes, is there also a chance to make it working in my screenshot ? so from the other way around, so that I can first select a category, and below get "all" posts ? (but not only the ones that are already connected to the category, instead "all", but with a virtual-column that shows, if the post is connected or not)
